package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "prepublish": "node ./apidocs/describe-builtin-models.js",
-    "pretest": "jshint .",
-    "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run --browsers PhantomJS"
+    "test": "node ./test.e2e/test-server.js node node_modules/karma/bin/karma start --single-run  --browsers PhantomJS",
+    "posttest": "npm run prepublish && eslint ."
```

Comment:
prepublish creates a local file `loopback-core.js` (aka `lb-services.js`), goal of this is to lint the generated code, because we don't lint the `ejs` file itself